packages/medulas-react-components/src/context/MatchMediaContext.tsx

```diff
@@ -0,0 +1,46 @@
+import { throttle } from 'lodash';
+import * as React from 'react';
+
+const phoneScreen = 425;
+
+type MatchMediaProps = boolean;
+
+const isPhone = (): boolean => (window.matchMedia(`(min-width: ${phoneScreen}px)`).matches ? false : true);
+
+export const MatchMediaContext = React.createContext<MatchMediaProps>(false);
```

Comment:
This was an ugly hack I created for moving forward, now we should rely on hooks, check the [mui solution](https://material-ui.com/components/use-media-query/#using-material-uis-breakpoint-helpers).

I have stomachache each time I see my code 😓

Reply:
Also I would like to see an example of storybook where we use this hook.

I am expecting to see something:
```
const isPhone = useMediaQuery(...);
....
```

Reply:
What to best to use? `theme.breakpoints` or hardcoded value for the phone screen width like now?

Reply:
bierzo wallet is not designed for the phone, you could chop out the responsive views and adaptation for the phone?

Reply:
ok